website/cue/reference/releases/0.47.0.cue

```diff
@@ -0,0 +1,427 @@
+
+releases: "0.47.0": {
+	date:     "2025-05-19"
+	codename: ""
+
+	whats_next: []
+
+	description: """
+		The Vector team is excited to announce version `0.47.0`!
+
+		Release highlights:
+
+		- The `opentelemetry` source now supports metrics ingestion.
+		- A new `window` transform has been introduced which enables log noise reduction by filtering out events during healthy state.
+		- A new `mqtt` source is now available, enabling ingestion from MQTT brokers.
+		- The `datadog_logs` sink now supports a new `conforms_as_agent` option to format logs like the Datadog Agent, ensuring compatibility with reserved fields.
+		"""
+
+	vrl_changelog: """
+		VRL was updated to `v0.24.0`. This includes the following changes:
+
+		#### Enhancements
+
+		- The `encode_gzip`, `decode_gzip`, `encode_zlib` and `decode_zlib` methods now uses the [zlib-rs](https://github.com/trifectatechfoundation/zlib-rs) backend
+			which is much faster than the previous backend `miniz_oxide`.
+
+		- The `decode_base64`, `encode_base64` and `decode_mime_q` functions now use the SIMD backend
+			which is faster than the previous backend.
+
+		#### Fixes
+
+		- Add BOM stripping logic to the parse_json function.
+		"""
+
+	changelog: [
+		{
+			type: "feat"
+			description: """
+				The `opentelemetry` source now supports metrics ingestion.
+				"""
+			contributors: ["cmcmacs"]
+		},
+		{
+			type: "feat"
+			description: """
+				Add a new `window` transform, a variant of ring buffer or backtrace logging implemented as a sliding window.
+				Allows for reduction of log volume by filtering out logs when the system is healthy, but preserving detailed
+				logs when they are most relevant.
+				"""
+			contributors: ["ilinas"]
+		},
+		{
+			type: "feat"
+			description: """
+				Adds a new `mqtt` source enabling Vector to receive logs from a MQTT broker.
+				"""
+			contributors: ["mladedav", "pront", "StormStake"]
+		},
+		{
+			type: "feat"
+			description: """
+				Add support for rendering to Mermaid format in `vector graph`
+				"""
+			contributors: ["Firehed"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fix a Vector crash that occurred when the internal metrics generated a lot of groups by increasing groups max limit from 128 to 256.
+				"""
+			contributors: ["triggerhappy17"]
+		},
+		{
+			type: "feat"
+			description: """
+				Allows users to specify AWS authentication and the AWS service name for HTTP sinks to support AWS API endpoints that require SigV4.
+				"""
+			contributors: ["johannesfloriangeiger"]
+		},
+		{
+			type: "feat"
+			description: """
+				Add support for fluentd forwarding over a unix socket
+				"""
+			contributors: ["tustvold"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Replaced miniz_oxide backend for zlib compression with zlib-rs for significantly improved compression performance. zlib-rs provides 2-3x performance improvements for compression, and ~20% improvements for decompression than miniz_oxide.
```

Comment:
Suggestion: write this as a user-facing change. They aren't going to know how the libraries impact their usage. I'd just mention that `zstd` compression and decompression are more efficient.